server/services/nl.py

```diff
@@ -58,31 +60,32 @@ def _download_embeddings(self):
       _, filename = os.path.split(blob.name)
       blob.download_to_filename(os.path.join(TEMP_DIR, filename))  # Download
 
-  def detect_svs(self, query):
+  def detect_svs(self, query, embeddings_build):
     query_embeddings = self.model.encode([query])
+    if embeddings_build not in self.dataset_embeddings_maps:
+      return {'SV': [], 'CosineScore': []}
     hits = semantic_search(query_embeddings,
-                           self.dataset_embeddings_maps[BUILD],
-                           top_k=15)
+                           self.dataset_embeddings_maps[embeddings_build],
+                           top_k=10)
```

Comment:
Just wanted to make sure that 15->10 is intentional. Please ignore this comment if it is!

Reply:
Yup, it was :)